### ISSUE TITLE
">" link isn't recognized by VoiceOver

### ISSUE THREAD

**Issue by RichMorin** [vjeantet/hugo-theme-docdock#157]
# Background

Sidebar entries have the form "> Foo". Both ">" and "Foo" are clickable links; ">" expands the entry, causing subsidiary elements to be displayed; "Foo" jumps the user to the "Foo" page.

# Problem

Although the "Foo" link is recognized and announced by VoiceOver (on Apple's macOS), the ">" link is not. So, this affordance is inaccessible to blind users.

**Comment by Pilskalns** [comment]
Hi @RichMorin,

I do not have any VoiceOver capable device in my possession nor do have any experience with such tools.
Is there any suggested solution how the HTML should be rendered to have usable experience?
